README.md

```diff
@@ -1,5 +1,7 @@
 #  resin-am571x-evm repository
 
+## This BSP has been discontinued and has no further updates planned from Resin.io.
```

Comment:
In this context I was using "Resin.io" in the context of "Resin.io the company" rather than the context of "Resin.io the product". I'm not sure what the best terms are to be specific. Do you have any updated wording in mind?